capstone/src/main/java/com/google/sps/servlets/clubs/EditClubServlet.java

```diff
@@ -75,10 +75,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
                           .replaceAll("\\s", "")) // Removes all whitespace and moves to lower case.
               .filter(Predicates.not(Strings::isNullOrEmpty))
               .collect(toImmutableList());
+      boolean isExclusive = request.getParameter(Constants.EXCLUSIVE_PROP) != null;
```

Comment:
Just to be clear, this works because it is not possible for `request.getParameter(Constants.EXCLUSIVE_PROP)` to return false right? It would only return something if the checkbox is checked?

Want to make sure this doesn't happen: `false != null` -> true

Reply:
Yeah, the HTML checkbox component only returns a String value ("on" if a value is not specified) if checked, or null if not checked.